### PR TITLE
feat: add intent for konnector installation ✨

### PR DESCRIPTION
**Pull request by gregorylegarec**
This PR :
* adds intent handling
* extract the `<AppInstallation />` component from `<InstallModal />` to use it in `<KonnectorInstallIntent />`

### CODE REVIEW

**Comment by gregorylegarec** [comment]
[(5) [COL] ETU je peux installer/configurer UN connecteur depuis de l'inter-apps](https://trello.com/c/PnUUxk1F/898-5-col-etu-je-peux-installer-configurer-un-connecteur-depuis-de-linter-apps)

File: manifest.webapp [hunk 1]
```diff
@@ -14,11 +14,26 @@
   },
   "default_locale": "en",
   "locales": {},
+  "intents": [
+    {
+      "action": "INSTALL",
+      "data": ["slug"],
+      "href": "/intents",
+      "type": [
+        "io.cozy.konnectors"
```
Comment (CPatchane) [inline manifest.webapp]: Should we use a generic `io.cozy.apps` like for the redirection? I mean we could need to install an application from the intent? Or is this completely different? I'd like to keep things generic between konnectors and webapps in the Store.

File: src/ducks/apps/components/AppInstallation.jsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+class AppInstallation extends Component {
+  installApp () {
+    this.setState({ error: null })
+    const { app, onSuccess } = this.props
+    this.props
+      .installApp(app.slug, app.type)
+      .then(() => {
+        onSuccess(app)
+      })
+      .catch()
```
Comment (CPatchane) [inline src/ducks/apps/components/AppInstallation.jsx]: We don't catch anything? Or is this caught by the Redux store? I thinks either we need it and we have to process the error either we don't need it at all here.

File: src/ducks/components/intents/InstallKonnectorIntent.jsx [hunk 3]
```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react'
```
Comment (CPatchane) [inline src/ducks/components/intents/InstallKonnectorIntent.jsx]: Again, has this component to be specific to konnector? Could we use a generic `InstallAppIntent` or this component is doing some things specific to konnectors?

File: src/ducks/components/intents/IntentHandler.jsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    }
+
+    props.intents
+      .createService()
+    // Free : easy mocking !
+    // Promise.resolve({
+    //   getData: () => ({ slug: <konnector slug> }),
+    //   getIntent: () => ({ action: 'INSTALL', type: 'io.cozy.konnectors' }),
+    //   terminate: (doc) => { alert(`Installed ${doc.name}`) }
+    // })
```
Comment (CPatchane) [inline src/ducks/components/intents/IntentHandler.jsx]: 👍

**Comment by CPatchane** [comment]
Thanks @gregorylegarec 🌜

**Pull request merged**